index.html

```diff
@@ -1,98 +1,47 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="styles.css" />
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
-  </head>
-  <body>
-    <form class="form__container" action="" id="inputvalorIngresadoDelUsuario">
-      <div>
-        <label class="form__label" for="name">
-          Nombre:
-          <input
-            class="form__input"
-            id="inputNombreDelUsuario"
-            type="text"
-            name="nombre"
-            placeholder="Ingrese su nombre"
-          />
-        </label>
-        <label class="form__label" for="name">
-          Apellido:
-          <input
-            class="form__input"
-            id="inputApellidoDelUsuario"
-            type="text"
-            name="apellido"
-            placeholder="Ingrese su apellido"
-          />
-        </label>
+    <!-- CSS only -->
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+<body>
+    <div class="container pt-3">
+        <h1>Calcular Impuesto</h1>
+        <form id="formImpuestos" action="">
+          <div class="mb-3">
+            <label for="Modelo" class="form-label">Coloque una cantidad de dinero "$"</label>
+            <input class="form-control" id="precio" type="number" placeholder="Ingrese la cantidad de dinero" />
+          </div>
+          <div class="mb-3">
+            <label for="Marca" class="form-label">Coloque el impuesto "%" que quiere aplicarle</label>
+            <input class="form-control" id="impuesto" type="number" placeholder="Ingrese el impuesto" />
+            <div id="errorValorImpuesto"></div>
+          </div>
+          <div class="mb-3">
+            <label for="Precio" class="form-label">Coloque el descuento "%" que quiere aplicar</label>
+            <input class="form-control" id="descuento" type="number" placeholder="Ingrese el descuento " />
+          </div>
+          <button class="btn btn-primary mb-3" id="enviarDatos">Enviar datos</button>
+        </form>
       </div>
+      <table class="table">
```

Comment:
Agreguemos un container para que tenga los mismos margenes que el form
```suggestion
   <div class ="container">
      <table class="table">
```